### ISSUE TITLE
Seeing NullRef exception from SystemHeatFissionEngineKerbalismUpdater

### ISSUE THREAD

**Issue by genehwung** [judicator/KerbalismSystemHeat#4]
Trying this out in KSP 1.12.2, seeing this in log:

```
[ERR 11:13:55.608] Module SystemHeatFissionEngineKerbalismUpdater threw during OnLoad: System.NullReferenceException: Object reference not set to an instance of an object
  at SystemHeat.ModuleUtils.GetModuleConfigNode (Part part, System.String moduleName) [0x0005b] in <3f2e635a95c64c898b6389032422a90f>:0 
  at KerbalismSystemHeat.SystemHeatFissionEngineKerbalismUpdater.ParseResourcesList (Part part) [0x0000b] in <dd4d75ec23ea4c9a90a6006789ede9f7>:0 
  at KerbalismSystemHeat.SystemHeatFissionEngineKerbalismUpdater.OnLoad (ConfigNode node) [0x0000e] in <dd4d75ec23ea4c9a90a6006789ede9f7>:0 
  at PartModule.Load (ConfigNode node) [0x001ab] in <cd473063d3a2482f8d93d388d0c95035>:0 
```

See the full KSP.log and player.log [here](https://drive.google.com/drive/folders/1GFeO7Y48UyGm2k_GgUJYsm6mqEnGlJgQ?usp=sharing).

I also see similar error for KerbalismFFT, probably the same root cause?

```
Module FFTFusionEngineKerbalismUpdater threw during OnLoad: System.NullReferenceException: Object reference not set to an instance of an object
  at SystemHeat.ModuleUtils.GetModuleConfigNode (Part part, System.String moduleName) [0x0005b] in <3f2e635a95c64c898b6389032422a90f>:0 
  at KerbalismFFT.FFTFusionEngineKerbalismUpdater.ParseModesList (Part part) [0x00008] in <b9efb20047d049b2a45ec3b46747cbf0>:0 
  at KerbalismFFT.FFTFusionEngineKerbalismUpdater.OnLoad (ConfigNode node) [0x0000e] in <b9efb20047d049b2a45ec3b46747cbf0>:0 
  at PartModule.Load (ConfigNode node) [0x001ab] in <cd473063d3a2482f8d93d388d0c95035>:0  
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)
```

**Comment by VeronikaKerman** [comment]
Also occurs on ksp 1.11.2 and systemheat 0.5.6.0

**Comment by VeronikaKerman** [comment]
When OnLoad is called, no part has yet been created (it's before main menu), so the GetModuleConfigNode fails. Nertea's code catches the exception, but then tries to print a message with the part's name. Could the resource parsing code be moved later in the loading process?

**Comment by judicator** [comment]
Fixed in 0.5.0.
